src/source/reference/delete-campaign.rst

```diff
@@ -0,0 +1,24 @@
+.. index:: DELETE /campaign/{id}
+.. _delete_campaign:
+
+DELETE /campaign/{id}
+===============================
+
+It is possible to remove a campaign in one go. This does not remove the ads from your inventory; it only takes them offline for that particular campaign.
```

Comment:
Does this set the campaign to DELETED only, or does it also remove the links?

Reply:
unexposed (and will be for a loong time I think, only for internal console testing), but it will remove the links.